src/components/search/search-result.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LevelIndicator from '../level-indicator/level-indicator';
+import ReactHtmlParser from 'react-html-parser';
+import Icon from '@mapbox/mr-ui/icon';
+
+class SearchResult extends React.Component {
+  returnRaw = item => {
+    if (item && item.raw && typeof item.raw !== 'string')
+      return item.raw.join(', ');
+    else return item && item.raw ? item.raw : '';
+  };
+
+  render() {
+    const { props } = this;
+
+    const getItemProps = props.downshiftProps.getItemProps;
+    const highlighted = props.downshiftProps.highlightedIndex === props.index;
+    const site = this.returnRaw(props.result.site);
+    const subsite = this.returnRaw(props.result.subsite);
+    const type = this.returnRaw(props.result.contentType);
+    const level = this.returnRaw(props.result.level);
+    const language = this.returnRaw(props.result.codeLanguage);
+    const title = this.returnRaw(props.result.title);
+    const url = this.returnRaw(props.result.url);
+    const excerpt = props.result.excerpt
+      ? props.result.excerpt.snippet || props.result.excerpt.raw
+      : '';
+    return (
+      <div
+        className="py12 px18"
+        {...getItemProps({
+          key: props.result.id.raw,
+          item: props.result,
+          className: `${highlighted &&
+            'bg-gray-faint'} py12 px18 link--gray cursor-pointer`
+        })}
+      >
+        {title && url && (
+          <div className="block link--gray">
+            <div className="mb3">
+              <span className="txt-bold">
+                {site && site !== title ? (
+                  <span>
+                    {site}
+                    <Icon name="chevron-right" inline={true} />
+                  </span>
+                ) : (
+                  ''
+                )}
+                {subsite && subsite !== title && subsite !== site ? (
+                  <span>
+                    {subsite.replace(/\sfor\s(iOS|Android|Vision)/, '')}
```

Comment:
💡This temporarily removes the "for Android" (or iOS or Vision) from the subsite name. We should eventually update the docs-page-shell to strip this out but this will fix it on the frontend to give us cleaner looking results.

Reply:
Good call! Much cleaner. Should we do the same for Unity? 

![image](https://user-images.githubusercontent.com/10479155/57548734-9521b600-7316-11e9-93b5-d5ec07a3a62c.png)